modules/EmailMan/EmailMan.php

```diff
@@ -624,19 +637,31 @@ public function sendEmail($mail, $save_emails = 1, $testmode = false)
             $GLOBALS['log']->fatal('Encountered invalid email address: ' . $module->email1 . " Emailman id=$this->id");
             return true;
         }
-        
-        if (
-                $module instanceof Basic &&
-                $sugar_config['email_enable_confirm_opt_in'] &&
-                $module->fromSugarEmailAddressField('email1')->confirm_opt_in
-        ) {
-            global $log;
-            $log->warn('An email is not confirmed opt in: '. $module->email1);
+
+        if ($this->shouldBlockEmail($module)) {
+            $GLOBALS['log']->debug('Email Address was sent due to not being confirm opt in' . $module->email1);
+
+            // block sending campaign email
             $this->set_as_sent($module->email1, true, null, null, 'blocked');
-            $success = false;
-        } elseif ((!isset($module->email_opt_out) || ($module->email_opt_out !== 'on' && $module->email_opt_out !== 1 && $module->email_opt_out !== '1')) && (!isset($module->invalid_email) || ($module->invalid_email !== 'on' && $module->invalid_email !== 1 && $module->invalid_email !== '1'))) {
+            return true;
```

Comment:
I believe this is correct. In EmailManDelivery.php:251, the return type determines what log message is printed. when the sendEmail returns true - the EmailManDelivery class will print a debug message  "Email delivery SUCCESS", when sendEmail returns false - the EmailManDelivery class will print a fatal message  "Email delivery FAILURE".

The reason I have changed the return type to true, is because the other code blocks in the sendEmail which block the email also return true. Which makes sense, because the delivery didn't fail, instead we blocked it.